File: Trabalhos/Ambientes desenv de software/Entregável #1 - Algoritmos/Fibonacci.js
var N

function Fibo(n) {
    for (i=1;i<=n;i++){
        function FiboInter(n){
        let antepNum
        let penNum = 0
        let numAtual = 1
        if ((n==0)||(n==1)){
            return n
        }
        for(x=2;x<=n;x++){
            antepNum = penNum
            penNum = numAtual
            numAtual = antepNum + penNum
        }
        return numAtual
        }
        console.log(FiboInter(i))
    }
}

Fibo(prompt("Insira quantos números da sequência você deseja ?"))

File: Trabalhos/Ambientes desenv de software/Entregável #1 - Algoritmos/Java/src/main/java/Fibonacci.java
import java.util.*;

public class Fibonacci {
  public void Fibo(int n) {
    List<Integer> fibonacci = new ArrayList<Integer>();
    for (int i = 1; i <= n; i++) {
      fibonacci.add(FiboInter(i));
    }
    System.out.println("Alg_Fibonacci: " + fibonacci + " são os números até a posição indicada");
  }

  public int FiboInter(int n) {
    int antepNum;
    int penNum = 0;
    int numAtual = 1;

    if (n == 0 || n == 1) {
      return n;
    }

    for (int x = 2; x <= n; x++) {
      antepNum = penNum;
      penNum = numAtual;
      numAtual = antepNum + penNum;
    }
    return numAtual;
  }

  public Fibonacci(int n) {
    Fibo(n);
  }
}